_includes/footer.html

```diff
@@ -3,26 +3,31 @@
         <div class="row">
             <div class="col-sm-12 col-md-9">
                 <h6>Acknowledgements</h6>
-                <img class="europe-flag" src="images/Flag_of_Europe.svg" alt="Company logo"/>
+                <img class="europe-flag" src="images/Flag_of_Europe.svg" alt="Company logo" />
                 <p class="text-justify">
                     <a href="https://elixir-europe.org/about-us/how-funded/eu-projects/converge">ELIXIR-CONVERGE</a><br>
-                    Contributions from the 23 ELIXIR Nodes
+                    Contributions from the 23 ELIXIR Nodes<br>
+                    <a href="mailto:{{ site.contact_email }}">{{ site.contact_email }}</a>
                 </p>
             </div>
             <div class="col-xs-12 col-md-3">
-                <img class="company-image" src="images/converge_logo_inverted.svg" alt="Company logo"/>
+                <img class="company-image" src="images/converge_logo_inverted.svg" alt="Company logo" />
             </div>
         </div>
     </div>
     <div class="container">
         <div class="row">
-            <div class="col-xs-12 col-md-6">
-                <p class="contact-us"><a href="mailto:{{ site.contact_email }}">{{ site.contact_email }}</a>
-                </p>
-            </div>
-            <div class="col-xs-12 col-md-6">
-                <p class="copyright-text">{{ site.time | date: "%Y"  }} {{site.company_name}}. All rights reserved.
-                </p>
+            <div class="col-xs-12 col-md-12 copyright-text">
+                <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License"
+                        style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br /><span
+                    xmlns:dct="http://purl.org/dc/terms/" property="dct:title">RDMKit</span> by <a
+                    xmlns:cc="http://creativecommons.org/ns#"
+                    href="https://elixir-europe.org/about-us/how-funded/eu-projects/converge"
+                    property="cc:attributionName" rel="cc:attributionURL">ELIXIR-CONVERGE</a> is licensed under a <a
+                    rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons
+                    Attribution 4.0 International License</a>, except where otherwise noted. All material under this
+                licence can be freely used, as long as <a rel="license"
+                    href="https://rdmkit.europe-belgium.org">RDMKit</a> is credited as the author.
```

Comment:
I wouldn't add explanatory text, it's safer to just provide the link.

```suggestion
                    Attribution 4.0 International License</a>, except where otherwise noted.
```